Formula/gettext.rb

```diff
@@ -5,6 +5,7 @@ class Gettext < Formula
   mirror "https://ftpmirror.gnu.org/gettext/gettext-0.21.tar.xz"
   sha256 "d20fcbb537e02dcf1383197ba05bd0734ef7bf5db06bdb241eb69b7d16b73192"
   license "GPL-3.0"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```
Headers in source tarball say `or (at your option) any later version`.